lib/elements/footing.rb

```diff
@@ -0,0 +1,35 @@
+module Elements
+  class Footing
+    attr_reader :concrete
+
+    attr_accessor :step, :dimension1, :dimension2, :height
+
+    def initialize(dimension1:, dimension2:, height:, concrete:, step: 100)
+      @dimension1 = dimension1.to_f
+      @dimension2 = dimension2.to_f
+      @height = height.to_f
+      @step = step.to_f
+      @concrete = concrete
+    end
+
+    def change_concrete_to(concrete)
+      @concrete = concrete
+    end
```

Comment:
how is this different from setting `attr_accessor :concrete`? I'm seeing you are doing that for all attributes except for `concrete`, but instead you are adding this method